conf/application.conf

```diff
@@ -70,6 +70,13 @@ webKnossos {
 
     Please add the information of the operator to comply with GDPR.
   """
+
+  OIDC {
+    providerURL = "http://localhost:8080/auth/realms/master/"
```

Comment:
I’d say this should all be empty strings in the repository, as this feature should be disabled by default. I’d say sample values for testing can be found in the PR description/discussion and this is enough, we should be able to find the PR again later.

Reply:
Agreed, since these are values for testing on my side. However, I think this could be done last. Should there also be a configuration option that disables OIDC entirely? Or should that be reflected by empty values? Deactivated OIDC should result in some different error messages on the routes.

Reply:
> I think this could be done last

fair :)

> Should there also be a configuration option that disables OIDC entirely? Or should that be reflected by empty values? Deactivated OIDC should result in some different error messages on the routes.

I’d say an empty providerUrl means oidc auth is disabled. Some of the config options have this semantic already. I agree that the routes should have an assertion on this and emit a readable error message then.